File: 1-TwoSum/1-TwoSum.cpp
// Last updated: 7/28/2025, 3:30:47 PM
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int i,j;
      int   n=nums.size();
        for(i=0;i<n-1;i++)
        {
            for(j=i+1;j<n;j++)
            {
                if(nums[i]+nums[j]==target){
                    return {i,j};
                }
            }
        }
        return{};
    }
};

File: 1-TwoSum/1-TwoSum.py
# Last updated: 8/7/2025, 11:08:40 AM
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        n=len(nums)
        for i in range(len(nums)):
            for j in range(i+1,n):
                if(nums[i]+nums[j]==target):
                    return (i,j)
               
     
        